q2cli/builtin/tools.py

```diff
@@ -80,40 +80,149 @@ def export_data(input_path, output_path, output_format):
     click.echo(CONFIG.cfg_style('success', success))
 
 
-def show_importable_types(ctx, param, value):
-    if not value or ctx.resilient_parsing:
-        return
+def print_descriptions(descriptions, tsv):
+    if tsv:
+        for value, description in descriptions.items():
+            click.echo(f"{value}\t", nl=False)
+            if description:
+                click.echo(deformat_description(description))
+            else:
+                click.echo()
+    else:
+        import textwrap
+        for value, description in descriptions.items():
+            click.secho(value, bold=True)
+            if description:
+                description = deformat_description(description)
+                tabsize = 8
+                wrapped_description = textwrap.wrap(description,
+                                                    width=72-tabsize,
+                                                    initial_indent='\t',
+                                                    subsequent_indent='\t',
+                                                    tabsize=tabsize)
+                for line in wrapped_description:
+                    click.echo(f"{line}")
+            else:
+                click.secho("\tNo description", italic=True)
+            click.echo()
+
+
+def deformat_description(description):
+    import re
```

Comment:
This import happens every time you call this function. I'm not sure if Python recognizes that and doesn't do the import after the first time, or if it's doing work to import on every call. You can probably instead throw this import in at the top of `print_descriptions`, since you're almost certainly going to call `deformat_description` from their.

Reply:
The overhead is so tiny (a dictionary miss) that it seems worth it for clarity's sake to keep it in the function that has the dependency, but I'm happy to move it too.

Reply:
I agree that this is an acceptable performance hit for the sake of code clarity.